software/myip.yml

```
@@ -0,0 +1,13 @@
+name: "MyIP"
+website_url: "https://ipcheck.ing"
+source_code_url: "https://github.com/jason5ng32/MyIP"
+description: "All in one IP Toolbox. Easy to check what's your IPs, IP geolocation, check for DNS leaks, examine WebRTC connections, speed test, ping test, MTR test, check website availability and more."
+licenses:
+  - MIT
+platforms:
+  - Nodejs
+tags:
+  - Network Tools
+  - Monitoring
```

Tag Network Tools doesn't exist, and Monitoring is a redirect and therefore can't be used.
```suggestion
tags:
  - Miscellaneous
```